dialogues/bitod/src/knowledgebase/en_zh_mappings.py

```diff
@@ -25,6 +26,13 @@ def read_require_slots():
     return require_slots
 
 
+risawoz_mapping = RisaWOZMapping()
```

Comment:
We need  the mapping for English too

Reply:
I've created a PR to revise your machine-translated mapping. Please let me know if you need other alignment information.